### PR TITLE
June Serenity updates

### PR DESCRIPTION
**Pull request by NolwennP**
**To be merged on the 2nd of July.**

Checklist before merging into master.
- [x] All screenshots are ok.
- [x] English has been approved.
- [x] Month description is ok.
- [x] Month illustration is ok.
- [x] Each feature/improvement has its metadata.
- [x] All the `more` and `meta-data` links are working.

### CODE REVIEW

File: content/updates/index.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+  "2020-06": {
+    "title": "☀️ June Serenity updates",
+    "description": "This are the June Serenity updates",
```
Comment (LaureBro) [inline content/updates/index.json:4]:
```suggestion
    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and get more flexibility on asset naming conventions",
```

Comment (camillebourjade) [reply content/updates/index.json:4]: and execute asset naming conventions more easily",

Comment (LaureBro) [reply content/updates/index.json:4]: replaced with "execute asset naming conventions with more flexibility" thanks!

File: content/updates/index.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 {
+  "2020-06": {
+    "title": "☀️ June Serenity updates",
+    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and execute asset naming conventions with more flexibility",
+",
```
Comment (NolwennP) [inline content/updates/index.json:5]:
```suggestion
```

File: content/updates/index.json [hunk 3]
```diff
@@ -1,4 +1,9 @@
 {
+  "2020-06": {
+    "title": "☀️ June Serenity updates",
+    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and execute asset naming conventions with more flexibility",
```
Comment (NolwennP) [inline content/updates/index.json:4]:
```suggestion
    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and execute asset naming conventions with more flexibility.",
```

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
```
Comment (NolwennP) [inline content/updates/2020-06/1-duplicate-a-product.md:2]:
```suggestion
::: meta-data type="New" features="Enrichment, Product creation" available="early June" link-to-doc="../articles/duplicate-a-product.html"
```

File: content/updates/2020-06/2-rules-updates.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Rules engine updates
+
+At Akeneo, we aim at delivering the best user experience. A good experience with our PIM goes with a good understanding of what is going on, especially when there are errors that you have to correct. The following improvements will enable you to better monitor rule executions, and easily correct errors that can occur in the rules engine when importing a YML file or when executing rules.
+
+## Monitor rules execution under the process tracker
+
+::: meta-data type="Improvement" features="Rules engine, Productivity" available="mid-June" link-to-doc="../articles/manage-your-rules.html#execute-rules.html"
```
Comment (NolwennP) [inline content/updates/2020-06/2-rules-updates.md:7]:
```suggestion
::: meta-data type="Improvement" features="Rules engine, Productivity" available="mid-June" link-to-doc="../articles/manage-your-rules.html#execute-rules"
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# New: execute asset naming conventions manually
+::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:2]:
```suggestion
::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention"
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+There are two ways to manually execute a naming convention:
+- for one specific asset, directly in the asset edit form. The action is visible under the `...`
+
+![Unitary execution](../img/unitary-execution.png)
+
+- for an asset family, under the `product link rules` tab of your asset family - the naming convention will be applied to all assets belonging to this family.
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:19]:
```suggestion
::: more
[What is an asset naming convention?](../assets-product-link-rules.html#focus-on-the-naming-convention)
```

File: content/updates/index.json [hunk 8]
```diff
@@ -4,6 +4,11 @@
     "description": "These are the updates that happened in July.",
     "img": "illus-july-20.svg"
   },
+  "2020-06": {
+    "title": "☀️ June Serenity updates",
+    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and execute asset naming conventions with more flexibility.",
```
Comment (NolwennP) [inline content/updates/index.json:9]:
```suggestion
    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with our new attribute options screen and our rules engine updates and execute asset naming conventions with more flexibility.",
```

File: content/updates/2020-06/4-manually-execute-naming-conventions-on-assets.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+Asset naming conventions populate asset attributes with values contained in the asset filename or asset code. Prior to this improvement, naming conventions were executed once at the asset creation step. 
+
+
+There are two ways to manually execute a naming convention:
+- for one specific asset, directly in the asset edit form. The action is visible under the `...`
+
+![Unitary execution](../img/unitary-execution.png)
+
+- for an asset family, under the `product link rules` tab of your asset family - the naming convention will be applied to all assets belonging to this family.
```
Comment (camillebourjade) [inline content/updates/2020-06/4-manually-execute-naming-conventions-on-assets.md:22]:
```suggestion
- for an asset family, under the `product link rules` tab of your asset family - the naming convention will be applied to all the assets belonging to this family.
```

**Pull request merged**